src/lib/web/RouterConfigurer.ts

```diff
@@ -101,6 +104,12 @@ export class RouterConfigurer {
                 response.render(handlingResult.view, handlingResult.model);
             }
         }
+        for (let i = this.interceptors.length - 1; i >= 0; i -= 1) {
+            let interceptor = this.interceptors[i];
+            if (_.isFunction(interceptor.afterCompletion)) {
+                await interceptor.afterCompletion(request, response);
+            }
+        }
```

Comment:
afterCompletion should be changed to be called on hook on response when it is resolved